sycl/plugins/cuda/pi_cuda.cpp

```diff
@@ -5893,6 +5893,44 @@ pi_result cuda_piextCommandBufferMemBufferCopyRect(
   return {};
 }
 
+pi_result cuda_piextCommandBufferMemBufferRead(
+    pi_ext_command_buffer CommandBuffer, pi_mem Buffer, size_t Offset,
+    size_t Size, void *Dst, pi_uint32 NumEventsInWaitList,
```

Comment:
NumSyncPointsInWaitList not events, and for other new functions

Reply:
Cuda stub has been removed from this PR